.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    # Workflow files stored in the
+    # default location of `.github/workflows`
+    directory: "/"
+    schedule:
+      interval: "daily"
+    pull-request-branch-name:
+      separator: "/"
+    assignees:
+      - "wmmc88"
+    reviewers:
+      - "wmmc88"
+
+  - package-ecosystem: "gitsubmodule"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    pull-request-branch-name:
+      separator: "/"
+    assignees:
+      - "wmmc88"
+    reviewers:
+      - "wmmc88"
```

Comment:
@azumnanji  @hamzaMahdi do you want to be added here? its just the pr assignee and reviewers the automatic pr will set.